iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/AffMatchingAffOrgQualityTest.java

```diff
@@ -253,6 +255,29 @@ private AffMatchingService createAffMatchingService() {
         
         affMatchingService.setAffOrgMatchers(of(docOrgRelationMatcher, mainSectionHashBucketMatcher, firstWordsHashBucketMatcher));
         
+        
+        AffMatchOrganizationAltNameFiller altNameFiller = createAffMatchOrganizationAltNameFiller();
+        affMatchingService.setAffMatchOrganizationAltNameFiller(altNameFiller);
+        
         return affMatchingService;
     }
+   
+   
+   private AffMatchOrganizationAltNameFiller createAffMatchOrganizationAltNameFiller() throws IOException {
+       
+       AffMatchOrganizationAltNameFiller altNameFiller = new AffMatchOrganizationAltNameFiller();
+       
+       List<String> altNamesCountryCodes = of(
```

Comment:
OK for defining resources as simple list - maybe it will be more readable if there will be no loop for creating classpath strings.
I would leave `universities_` prefix in files. It is not long and quite 'readable'. If I would see bunch of files named with just two letters my first thought would be: Those files are named with some two random letters. When filename is prefixed then I would think that they are not some random and my brain would caught quicker that those two letters stands for country code.